Framework/PythonInterface/plugins/algorithms/BASISReduction.py

```diff
@@ -130,10 +131,32 @@ def PyInit(self):
         self.declareProperty('GroupDetectors', 'None',
                              StringListValidator(grouping_type),
                              'Switch for grouping detectors')
+        #
+        #  Normalization selector
+        #
+        title_flux_normalization = 'Flux Normalization'
+        self.declareProperty('DoFluxNormalization', True,
+                             direction=Direction.Input,
+                             doc='Do we normalize data by incoming flux?')
```

Comment:
Should documentation be written using first person point of view?

Reply:
Blame my thesis adviser :smile: